config/dydx/oracle.json

```diff
@@ -431,7 +431,7 @@
           "reconnectTimeout": 2000000000,
           "maxQueries": 1,
           "atomic": true,
-          "url": "https://dydx-api.polkachu.com",
+          "url": "localhost:1317",
```

Comment:
Is this some arbitrary localhost port?

Reply:
no its the rest endpoint for cosmos chains.

Reply:
Should we not provide a legitimate public API to fallback to as a default?

Reply:
> its the rest endpoint for cosmos chains.

Gotcha. I suppose it's fine to assume you're running the node locally then.